src/NuGet.Core/NuGet.Configuration/PackageSourceMapping/PackageSourceMappingConfiguration.cs

```diff
@@ -16,70 +16,70 @@ public class PackageNamespacesConfiguration
+        /// Get package source names with matching prefix "term" from package source mapping section.
         /// </summary>
         /// <param name="term">Search term. Cannot be null, empty, or whitespace only. </param>
-        /// <returns>Package source names with matching prefix "term" from package namespaces.</returns>
+        /// <returns>Package source names with matching prefix "term" from package patterns.</returns>
         /// <exception cref="ArgumentException"> if <paramref name="term"/> is null, empty, or whitespace only.</exception>
         public IReadOnlyList<string> GetConfiguredPackageSources(string term)
         {
             return SearchTree.Value?.GetConfiguredPackageSources(term);
         }
 
-        internal PackageNamespacesConfiguration(Dictionary<string, IReadOnlyList<string>> namespaces)
+        internal PackageSourceMappingConfiguration(Dictionary<string, IReadOnlyList<string>> patterns)
         {
-            Namespaces = namespaces ?? throw new ArgumentNullException(nameof(namespaces));
-            AreNamespacesEnabled = Namespaces.Keys.Count > 0;
+            Patterns = patterns ?? throw new ArgumentNullException(nameof(patterns));
+            IsEnabled = Patterns.Keys.Count > 0;
             SearchTree = new Lazy<SearchTree>(() => GetSearchTree());
         }
 
         /// <summary>
-        /// Generates a <see cref="PackageNamespacesConfiguration"/> based on the settings object.
+        /// Generates a <see cref="PackageSourceMappingConfiguration"/> based on the settings object.
         /// </summary>
         /// <param name="settings">Search term. Cannot be null, empty, or whitespace only. </param>
-        /// <returns>A <see cref="PackageNamespacesConfiguration"/> based on the settings.</returns>
+        /// <returns>A <see cref="PackageSourceMappingConfiguration"/> based on the settings.</returns>
         /// <exception cref="ArgumentNullException"> if <paramref name="settings"/> is null.</exception>
-        public static PackageNamespacesConfiguration GetPackageNamespacesConfiguration(ISettings settings)
+        public static PackageSourceMappingConfiguration GetPackageSourceMappingConfiguration(ISettings settings)
         {
             if (settings == null)
             {
                 throw new ArgumentNullException(nameof(settings));
             }
 
-            var packageNamespacesProvider = new PackageNamespacesProvider(settings);
+            var packageSourceMappingProvider = new PackageSourceMappingProvider(settings);
 
-            var namespaces = new Dictionary<string, IReadOnlyList<string>>();
+            var patterns = new Dictionary<string, IReadOnlyList<string>>();
 
-            foreach (PackageNamespacesSourceItem packageSourceNamespaceItem in packageNamespacesProvider.GetPackageSourceNamespaces())
+            foreach (PackageSourceMappingSourceItem packageSourceNamespaceItem in packageSourceMappingProvider.GetPackageSourceMappingItems())
```

Comment:
Maybe `GetPackageSourceMappingPatterns` could be better description of what it does?

Reply:
`PackageSourceMappingProvider` is an internal type. I think `GetPackageSourceMappingItems` is more apt because it returns a list of type `PackageSourceMappingSourceItem`. Please let me know if you disagree.